docs/ddl-of-carbondata.md

```diff
@@ -681,24 +682,28 @@ Users can specify which columns to include and exclude for local dictionary gene
 
      **NOTE:** Drop Complex child column is not supported.
 
-   - ##### CHANGE DATA TYPE
+   - ##### RENAME COLUMN AND CHANGE DATATYPE
```

Comment:
Change this to `Change Column Name/Type`. Please go through the below link and try to align the documentation in that way if possible
https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-ChangeColumnName/Type/Position/Comment